src/Components/EditModal/EditModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { ChangeEvent } from 'react';
+import Select, { components } from 'react-select';
+import { Movie } from '../MovieCard';
+import css from './EditModal.less';
+
+export function EditModal(
```

Comment:
Usually, Add and Edit modal use the same component and the difference is that there is an initial value for Edit

Reply:
fixed